code/game/objects/items/weapons/storage/boxes.dm

```diff
@@ -1211,12 +1211,12 @@
 	name = "flares box"
 	desc = "A box full of flares."
 	foldable = FALSE
-	max_storage_space = 4
+	max_storage_space = 12
```

Comment:
You should make a new "large box" variant instead, e.g. the large produce box used in the kitchen.